apps/callscreen/style/conference_group_ui.css

```diff
@@ -1,3 +1,8 @@
+html[dir="rtl"] #group-call-details-list > section .additionalContactInfo {
+  text-align: right;
+  padding: 0 0 2rem 0;
+}
+
```

Comment:
nit: Please move this below the regular LTR selector; I think it's clearer to have the RTL rule below each corresponding LTR selector